dor/config.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import sqlalchemy
+from pydantic.dataclasses import dataclass
+
+@dataclass
+class DatabaseConfig:
+    host: str
+    user: str
+    password: str
+    database: str
+    test_database: str
+
+
+@dataclass
+class Config:
+    database: DatabaseConfig
```

Comment:
Open to thoughts on this approach to consolidating configuration. Happy to change or adjust course...